pitch.md

```diff
@@ -23,38 +89,174 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+The users should write their schedule in a text file or provide an .ical file
+(generated from Google calendar or apple calendar, for users who do not want 
+ or need the increased expressiveness and concision provided by my DSL, but
+ would still like to schedule with others who **do** use the DSL).
+
+Here are some sample descriptions that I am considering:
+
+```
+time zone: PST
+
+On Monday, Wednesday, Friday:
+  busy 9am - 11am
+  busy 12pm - 12:30pm
+  busy 3pm - 4pm
+
+On Thursday:
+  free one_hour_of(2pm - 5pm)
+  busy after(7pm)
+
+On Weekdays:
+  free noon-1pm
+  free 2pm - 4pm, badness=**
+  free 6pm - 7pm, badness=***
+
+```
+
+We can see several things about the syntax.  First: it is almost certainly
+destined to be an external DSL, because of its freedom of characters and
+complicated parsing (sometimes times have minutes attached, modifiers for 
+the time of day, potentially military time, and the use of unary asterisk
+counters to represent how undesirable that block would be).
+
+There will also be a file provided by the leader which controls the
+portion of the calendar year which the scheduling could potentially occur.
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+When a program runs, there are several steps taken. The first step is,
+for each input file, the program parses the provided information into the
+language's internal representation.
+
+The next step in the run of the program is to combine the internal
+representation data generated for each individual into a version of the
+internal representation which contains, for each possible time in the range
+of possible schedules.
+
+Finally, the program would use this internal representation to attempt to
+find a solution to the scheduling problem and output this data to the
+user (perhaps with an explanation of why it was selected and some of the
+second choices -- providing these extra bits of information isn't essential,
+these are more of stretch goals).
+
+One of the semantic troubles which could occur would be if a user put in
+rules which are inconsistent.  One simple example of this might not be
+an error at all. For example, if someone says they are available on Weekdays
+from 1pm to 6pm and then they say that they are busy from 4 to 5 on Tuesdays,
+we don't assume they are inconsistent, we rather assume that both statements
+are valid except for the block during which they are in conflict, for which
+we consider the more specific case.
+
+This is similar to how browsers will interpret CSS, some conflicts are allowed
```

Comment:
CSS hidden inheritance is one of the more difficult parts of the language since it is sometimes hard to determine what rule is overriding another. As you design your language and error checking, I would keep that in mind. Should a program be able to compile with conflicting specs? Do you take the last spec for conflicting ones? I see you've thought/written about this below, but think its worth spending some thought on.